warehouse/integrations/snowflake/snowflake.go

```diff
@@ -847,7 +847,9 @@ func (sf *Snowflake) LoadIdentityMappingsTable(ctx context.Context) error {
 // * the load table strategy is "append" mode
 // * the uploader says we can append
 func (sf *Snowflake) ShouldAppend() bool {
-	return sf.config.loadTableStrategy == loadTableStrategyAppendMode && sf.Uploader.CanAppend()
+	return sf.config.loadTableStrategy == loadTableStrategyAppendMode &&
+		sf.Uploader.CanAppend() &&
+		!sf.Warehouse.GetBoolDestinationConfig("enableDedup")
```

Comment:
Same question about the load table strategy @lvrach 

Also what happens if the [upload.CanAppend()](https://github.com/rudderlabs/rudder-server/blob/release/1.15.x/warehouse/router/upload.go#L724) returns `false` but the user enabled the dedup?